samples/tab-stage-view/csharp/TabInStageView/AppPackage/manifest.json

```diff
@@ -0,0 +1,58 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
```

Comment:
Could you please check if this feature needs Dev Preview manifest schema? If not, please change it to latest schema.

https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema

Reply:
changed to latest schema